haskey-mtl.cabal

```diff
@@ -29,7 +29,9 @@ library
     mtl                     >=2.1  && <3,
     transformers            >=0.3  && <1,
     haskey-btree            >=0.3 && <0.4,
-    haskey                  >=0.3 && <0.4
+    haskey                  >=0.3 && <0.4,
+    monad-control           >=1.0.2.0 && <1.0.2.4,
```

Comment:
Could explain the lower bound on `monad-control`? Would `1.0.1.0` work as a lower bound? (To support LTS 6)